tests/helpers/module-for-acceptance.js

```diff
@@ -0,0 +1,23 @@
+import { module } from 'qunit';
```

Comment:
it was created by ember-cli addon blueprint, not sure if this file was missed intentionally